Idoit.API.Client/CMDB/ObjectTypes/ObjectTypes.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Idoit.API.Client.CMDB.ObjectTypes.Response;
+
+namespace Idoit.API.Client.CMDB.ObjectTypes
+{
+    public class ObjectTypes
+    {
+        public ObjectTypes(Client myClient)
+        {
+            client = myClient;
+        }
+
+        Dictionary<string, object> parameter;
+        List<Result[]> response;
+        public Client client;
+        public List<Result[]> Read()
+        {
+            Task t = Task.Run(() => { Reading().Wait(); }); t.Wait();
+            return response;
+        }
+
+        async Task Reading()
```

Comment:
Why is this called "Reading" but inside the function values are added to a list?

Reply:
Because when the method is called all parameters should be available.

Reply:
Can you explain this perhaps?